packages/subgraph/scripts/getLiquidations.ts

```diff
@@ -311,15 +311,15 @@ const getLiquidatedStreams = async (
     // Get dictionaries for optimizations
     const superTokenSymbolToBoolDict = superTokensResponse.response.reduce(
         (acc, obj) => {
-            return { ...acc, [obj.underlyingToken.symbol.toLowerCase()]: true };
+            return { ...acc, [obj.underlyingToken?.symbol.toLowerCase()]: true };
         },
         {} as { [symbol: string]: boolean }
     );
 
     const superTokenSymbolToIdDict = superTokensResponse.response.reduce(
         (acc, obj) => {
             const { id, underlyingToken } = obj;
-            return { ...acc, [underlyingToken.symbol.toLowerCase()]: id };
+            return { ...acc, [underlyingToken?.symbol.toLowerCase()]: id };
```

Comment:
Probably not what you want: <img width="174" alt="image" src="https://user-images.githubusercontent.com/10894666/149364560-b6679640-3b3f-497f-8fe9-a051f4a4512a.png">

Reply:
@kasparkallas mr. mr. please look now